mlflow/java/client/src/main/java/org/mlflow/api/proto/ModelRegistry.java

```diff
@@ -883,6 +883,7 @@ public RegisteredModel parsePartialFrom(
     /**
      * <pre>
      * User that created this ``registered_model``
+     * NOTE: this field is not currently returned.
```

Comment:
I had to regenerate the protobuf files for some reason.

Reply:
I found the commit. It appears the protobuf was not regenerated
https://github.com/mlflow/mlflow/commit/88135f51a1192f654d28d4fb6736b3dff076ae1a